pkg/flag/cloud_flags.go

```diff
@@ -1,40 +1,75 @@
 package flag
 
-import "github.com/aquasecurity/trivy/pkg/cloud"
+const (
+	DefaultApiURL         = "https://api.trivy.dev"
+	DefaultTrivyServerURL = "https://scan.trivy.dev"
+)
 
 var (
 	CloudTokenFlag = Flag[string]{
-		Name:       "token",
-		ConfigName: "cloud.token",
+		Name:       "cloud-token",
+		ConfigName: "cloud.cloud-token",
```

Comment:
In the config file, the dot denotes nesting.
For example, cache.dir means:
```yaml
cache:
  dir: "/path/to/cache"
```
Therefore, we don’t need to duplicate cloud here.
```suggestion
		ConfigName: "cloud.token",
```